src/config/dynamic-theme-fixes.config

```diff
@@ -157,6 +157,24 @@ body > #root > #wrapper > div[class^='c'] > #main-content > div:first-child
 
 ================================
 
+*.vultr.com
+
+CSS
+.svg-banner-shape {
+    fill: var(--darkreader-neutral-background);
+}
+
+IGNORE INLINE STYLE
+.icon-ui*
```

Comment:
Or I've missed something important in the syntax of CSS, but this shouldn't work and especially not with our matching system. And will probably error dark reader into something like:
'.icon-ui*' is not a valid selector

Reply:
I intended to tell darkreader to not touch `.icon-ui` at all, so I'm selecting all its children. What if it is changed to `.icon-ui *` (with a space in-between)?